loc/templates/loc/letter-content.html

```diff
@@ -69,7 +68,28 @@ <h2>Available access dates</h2>
     </div>
   {% endif %}
 
+  <div class="jf-avoid-page-breaks-within">
+    <h2>Requirements</h2>
+    <p>
+      In order to keep my household safe, I request that anyone 
+      entering my home to make repairs follow sanitation guidelines 
+      from the Center for Disease Control (CDC), including ________.
+    </p>
+    <p>
+      Additionally, I request that you provide the name and contact information 
+      for any repair-worker assigned to my home 24 hours prior to their arrival.
+    </p>
+    {% if has_heat_issues %}
+      <p>
+        Please be advised that the lack of Heat and/or Hot Water constitutes an 
+        emergency under Housing Maintenance Code. Failure to address these repairs 
+        will result in my escalating this issue and filing an Emergency HP Action 
+        through the NYC Housing Court system.
+      </p>
+    {% endif %}
```

Comment:
Interesting--since the emergency HPA criteria just got expanded from heat/hot water to a total of 7 issues, I'm curious if this will change too?

Reply:
Johanna just wants to leave this as these two issues for now, but def something to circle back to!